src/pocketmine/item/Item.php

```diff
@@ -524,6 +542,27 @@ public function getEnchantment(int $id){
 
 		return null;
 	}
+	
+	/**
+	* @param int $id
+	* @param int $level
+	*/
+	public function removeEnchantment(int $id, int $level = null){
```

Comment:
Agree, level in the remove function is pointless. Just check for the level in hasEnchantment, then remove it by ID.